### ISSUE TITLE
Crash on run: The specified module could not be found (apngasm)

### ISSUE THREAD

**Issue by sindastra** [laggykiller/sticker-convert#110]
Hi, I installed the latest version (2.2.5) through the MSI package on Windows 11, and it doesn't run.

The previous version I had installed (2.1.6) through the ZIP works fine.

Here's the error message:

```
sticker-convert 2.2.5
Launching GUI...
Traceback (most recent call last):
  File "C:\PROGRA~1\STICKE~1\sticker-convert.py", line 5, in <module>
  File "C:\PROGRA~1\STICKE~1\sticker_convert\__main__.py", line 16, in main
  File "C:\PROGRA~1\STICKE~1\sticker_convert\gui.py", line 19, in <module sticker_convert.gui>
  File "C:\PROGRA~1\STICKE~1\sticker_convert\job.py", line 21, in <module sticker_convert.job>
  File "C:\PROGRA~1\STICKE~1\sticker_convert\uploaders\upload_signal.py", line 13, in <module sticker_convert.uploaders.upload_signal>
  File "C:\PROGRA~1\STICKE~1\sticker_convert\converter.py", line 9, in <module sticker_convert.converter>
ImportError: LoadLibraryExW 'C:\PROGRA~1\STICKE~1\apngasm_python\_apngasm_python.pyd' failed: The specified module could not be found.
```

**Comment by laggykiller** [comment]
Thank you for your report. A bug report has been opened upstream: https://github.com/Nuitka/Nuitka/issues/2674

**Comment by laggykiller** [comment]
While we wait for upstream fix, you may install v2.2.6

https://github.com/laggykiller/sticker-convert/releases/tag/v2.2.6

**Comment by sindastra** [comment]
Thank you! I just wanted to confirm that the v2.2.6 workaround works on my end, both for the ZIP and MSI versions! (: 

(And thank you for handling this so quickly, I'm excited to go convert some more stickers now!)

**Comment by laggykiller** [comment]
Seems like the problem is on apngasm-python, not nuitka

Looking into it

**Comment by laggykiller** [comment]
Properly fixed in v2.2.7: https://github.com/laggykiller/sticker-convert/releases/tag/v2.2.7

Closing.
